api/types.pb.go

```diff
@@ -75,6 +75,7 @@ const (
 	// The main purpose of this state is to free up resources associated with service tasks on
 	// unresponsive nodes without having to delete those tasks. This state is directly assigned
 	// to the task by the orchestrator.
+	TaskStateRemove   TaskState = 800
```

Comment:
Please move this above the comment for TaskStateOrphaned and add a comment for this state.